lib/Entity/DataSet.php

```diff
@@ -492,6 +492,52 @@ public function getData($filterBy = [], $options = [])
             'connection' => 'default'
         ], $options);
 
+        // Fetch display tag value/s
+        if ($filter != '' && $displayId != 0) {
+            $displayTags = [];
+
+            // Define the regular expression to match [Tag:tagNameHere]
+            $pattern = '/\[Tag:(\w+)\]/';
```

Comment:
Does this work with all allowed tag names? I.e. if I have a tag called `$storeNo` ?

Reply:
I've replaced it now with a more generic regular expression to match `[Tag:...]` instead.